dev-manuals/architecture/template.md

```diff
@@ -0,0 +1,38 @@
+# Architecture of `$SERVICE`
+
+## Purpose
+
+`$PURPOSE`, i.e.
+```md
+Being a fun-to-play minigame that lets users choose one correct answer from a bunch of wrong answers
+```
+
+## General code structure
+
+`$STRUCTURE`,
+i.e.
+```md
+This backend has the following package structure
+- `data`: Contains the data objects and DTOs
+- `controllers`: Containers the server routes
+- `services`: Contains the business logic
+```
+
+## Starting the service
+
+See the README or the user manual.
+
+## Known Design Flaws
+
+- `$LIST OF DESIGN FLAWS PRESENT AT THE MOMENT`
+
+## Known Bugs
+
+An up-to-date list of open bugs can be found at <https://github.com/orgs/Gamify-IT/projects/6/views/11> by searching for `$SERVICE`.
```

Comment:
not sure if giving the project view is the best possibility to show known bugs. but atm i cant think of a better alternative

Reply:
The problem is that the board also has drafts, which didn't get to be an issue yet.
Alternatively, I can also link to https://github.com/orgs/Gamify-IT/issues/issues directly.
Would that be better?

Reply:
> The problem is that the board also has drafts, which didn't get to be an issue yet. Alternatively, I can also link to https://github.com/orgs/Gamify-IT/issues/issues directly. Would that be better?

i dont know ^^ i think if someone else would share their opinion on this would help

Reply:
Maybe we can add a direct link to the project view with the included query. For example for the overworld [`https://github.com/orgs/Gamify-IT/projects/6/views/1?filterQuery=-status%3AAccepted+-status%3ACancelled++label%3Abug+overworld`](https://github.com/orgs/Gamify-IT/projects/6/views/1?filterQuery=-status%3AAccepted+-status%3ACancelled++label%3Abug+overworld)

For another service we only need to replace the `overworld` at the end of the link

Reply:
I somewhat agree if we leave out the `+label%3Abug`, as I don't think we should only filter for bugs by default.
However, the problem is: For now, only minigames really follow the `$Minigame: Do XY` naming.
Should we require that for all issues?

Reply:
> For now, only minigames really follow the $Minigame: Do XY naming.
Should we require that for all issues?

Hmm thats a good question but currently your description also asks the user to do exactly that.

>  leave out the +label%3Abug

The section is called `Known bugs` so I don't know if it's a good idea to show issues about new features by default